src/core/Akka.Remote/Transport/TestTransport.cs

```diff
@@ -175,10 +173,13 @@ public Task Disassociate(TestAssociationHandle handle)
         public Task DefaultDisassociate(TestAssociationHandle handle)
         {
             var handlers = _registry.DeregisterAssociation(handle.Key);
-            handlers.Item1.Notify(new Disassociated(DisassociateInfo.Unknown));
-            handlers.Item2.Notify(new Disassociated(DisassociateInfo.Unknown));
+            if (handlers != null)
```

Comment:
Added a null-check here, which is the best C# translation I have for the `.forEach` call that Scala uses to ensure that the `Notify` handlers aren't called in the event that the `TestTransport` association has already been disassociated from the other side of the connection.